web/src/app/admin/admin-service-metrics/AdminServiceMetrics.tsx

```diff
@@ -180,7 +181,7 @@ export default function AdminServiceMetrics(): JSX.Element {
     return (
       <React.Fragment>
         <Grid item xs>
-          <Card sx={{ marginTop: (theme) => theme.spacing(1) }}>
+          <Card sx={{ marginTop: (theme: Theme) => theme.spacing(1) }}>
```

Comment:
Was your IDE complaining about this? It has always been fine (I assume the type has been inferred) on my end

Reply:
Well it was but it doesn't seem to be complaining anymore so reverted the updated.